app/connectors/ArrivalMovementConnector.scala

```diff
@@ -37,12 +37,26 @@ class ArrivalMovementConnector @Inject() (config: FrontendAppConfig, http: HttpC
 
   private val channel: String = "web"
 
-  def getArrivals()(implicit hc: HeaderCarrier): Future[Option[Arrivals]] = {
+  private def makeQueryParam(param: String, valueParam: Option[String]): Seq[(String, String)] =
+    valueParam.fold(Seq.empty[(String, String)])(
+      value => Seq((param, value))
+    )
+
+  def getArrivals(
+    mrn: Option[String] = None,
+    page: Option[String] = None,
```

Comment:
Hmmmm, I don't think the API allow for mrn and page at the same time. Is there something we can do to eliminate this as a possible case?

Reply:
Have two APIs? :) Well have to consider that possibly in the controller as well, if we share the same controller with different query params. (I did consider the case of both as well, but I wasn't sure if there ever would be the possibility of both in the future, if the requirements changed. Not that you want to code that way necessarily, but I wasn't sure if wanted to explicitly prevent it or not at this level.)